### DISCUSSION TITLE
FIT vs TCX vs JSON - Best Option?

### DISCUSSION THREAD

**Discussion by xavier4or** [philosowaffle/peloton-to-garmin#596] [Q&A] [answered]
Hi - newbie question - any metadata differences between the FIT, TCX, and JSON conversion?
Just trying to determine if there is a recommendation option?
Thanks!

**Comment by philosowaffle** [comment, marked as answer]
Hello! FIT is the best option and will save the most data.  TCX and JSON are there for people who have a specific need to use one of those formats.

**Reply by geoffbon** [reply]
I initially wondered this too.  Maybe worth changing the label in the UI to something like "FIT (recommended)"?

Edit: I see you have this in the help popup, so perhaps add the same help popup to the Garmin settings page?

**Reply by philosowaffle** [reply]
Yeah those help popouts are something new I'm testing out. Do plan to get them added to the other settings pages to help with this and I'll be sure to call out Fit is recommended.
